kitchen-terraform.gemspec

```diff
@@ -74,7 +74,10 @@ require "kitchen/terraform/version.rb"
     )
 
   specification.add_runtime_dependency "kitchen-inspec", "~> 0.18"
-
+  specification.add_runtime_dependency "awspec",  "~> 1.2"
+  
+  specification.add_runtime_dependency "kitchen-verifier-awspec", "~>0.1"
```

Comment:
While using a verifier other than the Kitchen-Terraform verifier is technically possible, it is not recommended. The Kitchen-Terraform verifier supports testing groups of servers from a Terraform state as well testing provider APIs.

Reply:
Now that I've seen how I can use your inspec verifier with awspec from your great docs (https://newcontext-oss.github.io/kitchen-terraform/tutorials/amazon_provider_ec2_advance.html) I don't know why I had that in there